docs/Shaders/Spinner2.md

```diff
@@ -0,0 +1,83 @@
+# Spinner2
```

Comment:
Should we mention something about Spinner (the previous version) and that it's recommended to use Spinner2?